.github/actions/build-image/action.yml

```diff
@@ -0,0 +1,38 @@
+name: 'Build images'
+description: 'Build images and push to ACR'
+inputs:
+  azure-credentials:
```

Comment:
Consider `.github/actions/build-image-nginx-otel/action.yml`. This has the name `nginx` in it. On the other hand, consider `.github/actions/build-image/action.yml`. This does not have any app name in it. I assume it is building the piggymetrics app. If that is true, perhaps a better name would be `.github/actions/build-piggymetrics-image/action.yml`.

Reply:
Good point. Fixed in 6898e41 with a different naming `build-image-piggymetrics` for consistency with the current naming pattern `build-image-nginx-otel`.